test/api.test.js

```diff
@@ -4,10 +4,13 @@ const fixtures = require('./fixtures/api.fixtures');
 const RO = require('..');
 const { generateBasicAuthToken } = require('../lib/utils/auth');
 const emitter = require('../lib/emitter');
+const { mockConfig } = require('../test/test-helpers/mock-config');
+
+const defaultConfig = mockConfig({ apiVersion: 'v4' });
 
 describe('api', () => {
   beforeAll(() => {
-    RO.config.set('apiVersion', 'v4');
+    RO.config.init(defaultConfig);
   });
```

Comment:
These tests need some clean up. They are threateningly un-atomic, which worries me re: bugs. Can we fix them in a follow-up ticket?

Reply:
Created ticket: https://rewardops.atlassian.net/browse/MX-1207

Can you take a look? And add comments about this test file? I am not 100% clear on what you mean by `un-atomic`

Reply:
Here's a decent article on atomic tests: https://techbeacon.com/app-dev-testing/how-make-your-test-automation-run-parallel